### DISCUSSION TITLE
unexplained crash

### DISCUSSION THREAD

**Discussion by akashpal-21** [minad/vertico#418] [Q&A] [answered]
```
Error during redisplay: ((lambda (win) (let ((height (window-height win))) (if (/= (1- height) vertico-count) (progn (set (make-local-variable 'vertico-count) (1- height)) (vertico--exhibit))))) #<window 7 on  *Minibuf-1*>) signaled (wrong-type-argument overlayp nil)
Error in post-command-hook (vertico--exhibit): (arith-error)
```

Sometimes while executing #'switch-to-buffer which is set to :reverse indexed grid unobtrusive , this message pops up and my vertico shell close to crashes, all the other commands get set to the same form and it becomes unusable, have to restart entire emacs session.

any advice?

**Comment by minad** [comment]
Hi, please provide a minimal recipe demonstrating the issue. A configuration `reverse indexed grid unobtrusive` does not make much sense, since some of these modes are mutually exclusive. Try enabling fewer modes. See also https://github.com/minad/vertico#debugging-vertico which will help you to obtain a full stack trace.

**Reply by akashpal-21** [reply]
Yes sir, I set it up some time back when I was very tired and it seemed to do what I wanted so didnt bother changing. The crash is very random, ill log it and post more information when it happens agains (if) in the mean time I will make the configuration sensible. 

Thank you for your response. 
Best,

**Reply by akashpal-21** [reply]
![Screenshot_2023-12-06_18-29-09](https://github.com/minad/vertico/assets/46517170/eec0d2cd-bb15-4d76-bea3-6b6828f6c495)

I can't figure out how to make it look like this by disabling any of these modes 😭 They seem to all contribute in creating this form. Although I know I shouldn't be using so many of these modes together 😭

**Reply by minad** [reply, marked as answer]
I think what you want is `flat` or `unobtrusive` and nothing else. Adding indexed on top doesn't add much value if only few candidates are at display.

**Reply by akashpal-21** [reply]
I totally forgot about the flat mode 🧑‍🎓 👍🏼 Thank you so much for the advice. I just went for the flat mode, its perfect for this use case.

**Reply by minad** [reply]
Indeed, it is. I use `flat` myself for files, but for M-x and buffers I went with the even more compact `unobtrusive`. And the vertical display is just a M-a away in my setup, via the `vertico-multiform-vertical` toggle command.
